electron/legendary/games.ts

```diff
@@ -665,10 +665,7 @@ class LegendaryGame extends Game {
     )
     appendFileSync(
       this.logFileLocation,
-      `Launch Command: ${fullCommand}
-
-Game Log:
-`
+      `Launch Command: ${fullCommand}\n\nGame Log:\n`
```

Comment:
```suggestion
    `Launch Command: ${fullCommand}\n` + 
          '\n' + 
          'Game Log:\n'
```

Reply:
Linting automatically changes this to `` `Launch Command: ${fullCommand}\n` + '\n' + 'Game Log:\n' ``